examples/reference/README.md

```diff
@@ -0,0 +1,303 @@
+Building a Stateful DC/OS Service
```

Comment:
I would delete this whole file, or replace it with a single reference to the new tutorial doc.  This is out of date.